Serilog.Sinks.InfluxDB/Serilog.Sinks.InfluxDB.csproj

```diff
@@ -13,8 +13,11 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <RepositoryUrl>https://github.com/pada57/serilog-sinks-influxdb</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <Version>1.4.1</Version>    
+    <Version>2.0.0-alpha.1</Version>    
```

Comment:
Isn't that very confusing? shouldn't we stay with 1.x for Influx 1.x, instead of introducing a new package with different name?